.github/workflows/pull_request_review.yml

```diff
@@ -16,12 +17,20 @@ jobs:
 
     strategy:
       matrix:
-        node: [10.x]
+        node: [12.x]
 
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - name: Ruby Gem cache
+      uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
```

Comment:
missing hash for restore ?

Reply:
I stole this from here since we're using Ubuntu: https://github.com/actions/cache/blob/main/examples.md#macos-and-ubuntu

Reply:
I just copied from here: https://github.com/actions/cache/blob/main/examples.md#ruby---bundler

I see cache hits in recent runs.